packages/unplugin/test/getReleaseName.test.ts

```diff
@@ -0,0 +1,98 @@
+import { getReleaseName } from "../src/getReleaseName";
+import * as fs from "fs";
+import * as child_process from "child_process";
+jest.mock("fs");
+jest.mock("child_process");
+
+const mockedFs = fs;
+const mockedChildProcess = child_process;
+
+describe("environmental getReleaseName", () => {
+  const OLD_ENV = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...OLD_ENV };
+  });
+
+  afterEach(() => {
+    (mockedChildProcess.execSync as jest.Mock).mockRestore();
+  });
+
+  afterAll(() => {
+    process.env = OLD_ENV;
+  });
+
+  it("adheres to HEAD when git is present", () => {
+    (mockedFs.existsSync as jest.Mock).mockReturnValueOnce(true);
+    const sha = "c3f235fc86f1c4007e3a218ec82d666586e73cbf";
+    (mockedChildProcess.execSync as jest.Mock).mockReturnValue(sha);
+
+    expect(getReleaseName()).toBe(sha);
+  });
+
+  it("throws an error if git is not present", () => {
```

Comment:
nit: I think this better describes the behaviour because this is the "last resort" if we have nothing to determine the release.
```suggestion
  it("throws an error if no release information could be found", () => {
```

Reply:
And since it's hackweek: Feel free to ignore ;)